Detectors/MUON/MID/Calibration/include/MIDCalibration/ChannelCalibrator.h

```diff
@@ -0,0 +1,114 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
+// See https://alice-o2.web.cern.ch/copyright for details of the copyright holders.
+// All rights not expressly granted are reserved.
+//
+// This software is distributed under the terms of the GNU General Public
+// License v3 (GPL Version 3), copied verbatim in the file "COPYING".
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file   MIDCalibration/ChannelCalibrator.h
+/// \brief  MID noise and dead channels calibrator
+/// \author Diego Stocco <Diego.Stocco at cern.ch>
+/// \date   18 February 2022
+
+#ifndef O2_MID_ChannelCalibrator_H
```

Comment:
that's a very minor thing, but according to https://rawgit.com/AliceO2Group/CodingGuidelines/master/coding_guidelines.html?showone=The__define_Guard#The__define_Guard I'd say the include guard should be uppercase.

Reply:
Fixed in the new version. I force pushed to keep the commits.